demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -0,0 +1,87 @@
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+from pylint.lint import PyLinter
+
+
+class CustomBaseChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+    name = "base-checker"
+    priority = -1
+    msgs = {
+        "E9001": (
+            # displayed-message shown to user
+            "Sys.exit use is found, Please use return instead.",
+            # message-symbol used as alias for message-id
+            "sys-exit-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Ensure to not use sys.exit in the code.",
+        ),
+        "E9002": (
+            # displayed-message shown to user
+            "Print is found, Please remove all prints from the code.",
+            # message-symbol used as alias for message-id
+            "print-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Please remove all prints from the code.",
+        ),
+        "E9003": (
+            # displayed-message shown to user
+            "demisto.log is found, Please remove, you can choose to replace it with demisto.debug or logger",
+            # message-symbol used as alias for message-id
+            "demisto-log-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Please remove all demisto.log from the code.",
+        ),
+        "E9004": (
+            # displayed-message shown to user
+            "time.sleep is found, Please remove any sleep functionality from the code",
+            # message-symbol used as alias for message-id
+            "sleep-exists",
+            # message-help shown to user when calling pylint --help-msg
+            "Please remove all sleep functionality from the code.",
+        )
+    }
+
+    def __init__(self, linter: PyLinter = None):
+        super(CustomBaseChecker, self).__init__(linter)
```

Comment:
no need to specify those in python3
```suggestion
        super().__init__(linter)
```

Reply:
As lint runs on dockers with both python2 and python3 , this is needed to enable the run on both.